src/main/java/woowacourse/auth/application/AuthService.java

```diff
@@ -1,7 +1,28 @@
 package woowacourse.auth.application;
 
 import org.springframework.stereotype.Service;
+import woowacourse.auth.domain.Customer;
+import woowacourse.auth.dto.token.TokenRequest;
+import woowacourse.auth.dto.token.TokenResponse;
+import woowacourse.auth.exception.InvalidAuthException;
+import woowacourse.auth.support.JwtTokenProvider;
 
 @Service
 public class AuthService {
+
+    private final CustomerService customerService;
+    private final JwtTokenProvider tokenProvider;
+
+    public AuthService(CustomerService customerService, JwtTokenProvider tokenProvider) {
+        this.customerService = customerService;
+        this.tokenProvider = tokenProvider;
+    }
+
+    public TokenResponse login(TokenRequest tokenRequest) {
+        Customer customer = customerService.findByEmail(tokenRequest.getEmail());
+        if (!customer.isSamePassword(tokenRequest.getPassword())) {
+            throw new InvalidAuthException("비밀번호가 일치하지 않습니다.");
+        }
+        return new TokenResponse(customer.getNickname(), tokenProvider.createToken(customer.getEmail()));
```

Comment:
```suggestion
        if (customer.isSamePassword(tokenRequest.getPassword())) {
            return new TokenResponse(customer.getNickname(), tokenProvider.createToken(customer.getEmail()));
        }
        throw new InvalidAuthException("비밀번호가 일치하지 않습니다.");
```

부정연산자는 가독성을 저하시키기 때문에 최대한 지양하는 방향을 고려하는 것이 좋습니다 :)

Reply:
앗 그렇네요 감사합니다!!